src/Estimators/src/FloatingBaseEstimator.cpp

```diff
@@ -37,25 +37,25 @@ bool FloatingBaseEstimator::initialize(std::weak_ptr<BipedalLocomotion::Paramete
 
 bool FloatingBaseEstimator::initialize(std::weak_ptr<BipedalLocomotion::ParametersHandler::IParametersHandler> handler)
 {
-    if (!m_modelComp.isKinDynValid())
+    auto handle = handler.lock();
+    if (handle == nullptr)
     {
-        std::cerr << "[FloatingBaseEstimator::initialize] The kindyn object with valid does not seem to be loaded."
-        << "Please call initialize(handler, kindyncomputations) to set the kindyn object."
+        std::cerr << "[FloatingBaseEstimator::initialize] The parameter handler has expired. Please check its scope."
         << std::endl;
         return false;
     }
 
-    if (m_estimatorState != State::NotInitialized)
+    if (!handle->getParameter("use_model_info", m_useModelInfo))
     {
-        std::cerr << "[FloatingBaseEstimator::initialize] The estimator already seems to be initialized."
-        << std::endl;
-        return false;
+        std::cerr << "[FloatingBaseEstimator::initialize] " <<
+                    "The parameter handler could not find \" use_model_info \" in the configuration file." <<
+                    "Setting to default value " << m_useModelInfo << std::endl;
+        m_useModelInfo = true;
     }
 
-    auto handle = handler.lock();
-    if (handle == nullptr)
+    if (m_estimatorState != State::NotInitialized)
     {
-        std::cerr << "[FloatingBaseEstimator::initialize] The parameter handler has expired. Please check its scope."
+        std::cerr << "[FloatingBaseEstimator::initialize] The estimator already seems to be initialized."
```

Comment:
```suggestion
        std::cerr << "[FloatingBaseEstimator::initialize] The estimator seems to be already initialized."
```

Reply:
Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/254/commits/10e3009d2a596e5fe5285718b91c776db4683086